admin/src/components/Wysiwyg/wrapper.js

```diff
@@ -2,13 +2,16 @@ import styled from 'styled-components';
 import { Box } from "@strapi/design-system/Box";
 
 const Wrapper = styled(Box)`
+    padding-bottom: 2.8rem;
```

Comment:
It is not entirely clear why such an indent is needed, Strapi has a lot of its own tools for solving problems with indents between fields. In what situation do we need this line?

Reply:
I agree. This was also in the original code. Can definitely be removed.